core/src/worker/utils.ts

```diff
@@ -104,9 +115,35 @@ export async function fetchDataWithAdapter(adapter, logger?: Logger) {
   const aggregatedResults = localAggregatorFn(...filteredResults)
   logger?.debug({ name: 'fetchDataWithAdapter', ...aggregatedResults }, 'aggregatedResults')
 
+  if (STORE_ADAPTER_FETCH_RESULT) {
+    for (const k in adapter) {
+      const a = adapter[k]
+      writeData(round, a.url, allResults[k])
+    }
+    writeData(round, 'AggregatedResult', aggregatedResults)
+  }
   return aggregatedResults
 }
 
+function writeData(round, url, data) {
+  const exportData = {
+    round: round,
+    url: url,
+    data: data,
+    time: new Date().getTime()
+  }
+  let row
+  const filename = path.join('src/cli', 'fetchHistory.csv')
```

Comment:
```suggestion
  const filename = path.join(LOG_DIR, 'fetchHistory.csv')
```

I think it should also include name of adapter, not just `fetchHistory.csv`.